.github/workflows/gradle.yml

```diff
@@ -28,12 +39,12 @@ jobs:
         run: ./runtest.sh
 
       - name: Perform IO redirection test (MacOS)
-        if: runner.os == 'macOS'
+        if: always() && runner.os == 'macOS'
```

Comment:
Does this mean the task will run even if it fails on the other two OS?

Reply:
Yes. There's a `continue-on-error` configuration but it causes errors to be marked with a green check so this is a workaround suggested by the github actions team.